packages/react-native-views/src/components/Thumbnail/ComplianceModal/index.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, StyleSheet } from 'react-native';
+import { Modal, Card, Button, Portal, withTheme } from 'react-native-paper';
+import { startCase, noop } from 'lodash';
```

Comment:
Can you please use one line per lodash package ?
```javascript
import startCase from 'lodash.startcase';
import noop from 'lodash.nopp';
```

It's because we only add the package we want on the project and this is clearer we don't have the all lodash mono repo install in modules.

Reply:
Yes ✅